packaging/vcpkg/CMakeLists.txt

```diff
@@ -48,5 +48,10 @@ MESSAGE(STATUS "using vcpkg target ${NOF_VCPKG_TARGET} with ${NOF_VCPKG_GIT_URL}
 set(NOF_VCPKG_PORT_FILE_OUT ${CMAKE_CURRENT_LIST_DIR}/ports/nearobject-framework/portfile.cmake)
 set(NOF_VCPKG_PORT_FILE_IN ${NOF_VCPKG_PORT_FILE_OUT}.in)
 
+# Delete the timestamp file
+set(SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE ${CMAKE_SOURCE_DIR}/windows/drivers/uwb/simulator/vcpkg_installed/x64-windows/.msbuildstamp-x64-windows.stamp)
+message(STATUS "removing simulator driver timestamp file: ${SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE}")
+file(REMOVE ${SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE})

```

Comment:
Would like to see a few small changes here:

1. Rename the variable with the file name to match the naming we use for the driver in the top-level `CMakeLists.txt` file ("WINDOWS_SIMULATOR_DRIVER").
2. Introduce an additional variable for the platform string used in the file name. This will be hard-coded for now, but will make it easier to update later.
3. Add a comment indicating the platform string is hardcoded.
4. Refer to the timestamp file as the vcpkg timestamp file to make clear this is vcpkg related.

```suggestion
# Delete the vcpkg timestamp file

# VS uses the MSBuild platform value ($(Platform)) to name the vcpkg timestamp file. This is not easy to obtain from CMake, so it is hardcoded for now, meaning this will only work on amd64/x64 architectures.
set(NOF_WINDOWS_PLATFORM "x64")
set(WINDOWS_UWB_SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE ${CMAKE_SOURCE_DIR}/windows/drivers/uwb/simulator/vcpkg_installed/${NOF_WINDOWS_PLATFORM}-windows/.msbuildstamp-${NOF_WINDOWS_PLATFORM}-windows.stamp)
message(STATUS "removing Windows simulator driver vcpkg timestamp file: ${WINDOWS_UWB_SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE }")

file(REMOVE ${WINDOWS_UWB_SIMULATOR_DRIVER_VCPKG_TIMESTAMP_FILE })

```